backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ClientProvider.java

```diff
@@ -94,7 +94,7 @@ private void configureSSLOptions(HttpAsyncClientBuilder builder) {
             try {
                 builder
                     .setSSLContext(sslContext())
-                    .setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE);
+                    .setSSLHostnameVerifier(ACCEPT_ANY_HOST);
```

Comment:
why are you still using a noop hostname verifier? Why not a real one, such as DefaultHostnameVerifier ?

Reply:
sorry, I missed understand you yesterday